pkg/build/pipelines/python/build-wheel.yaml

```diff
@@ -0,0 +1,17 @@
+name: Build a Python wheel
```

Comment:
Could we add some comments here on when one would use which variant? Even if it's heuristics?

Reply:
I would, but I don't actually know myself. There seem to be 80 ways of packaging stuff in Python, and this is just a few of them. Each project seems to do it a slightly different and fun way.